### ISSUE TITLE
how to transfer Spotify's "Your Music"?

### ISSUE THREAD

**Issue by benissimo** [rckclmbr/pyportify#48]
Pyportify lets one transfer Spotify playlists. But Spotify also lets us bookmarks tracks in a library named "Your Music" (with the nasty gotcha of a hidden limit of 10k tracks, hence my exodus...). Is there any way to transfer those as well?

**Comment by stevenbraham** [comment]
Just put all your music from "Your music" in one huge playlist :)

**Comment by benissimo** [comment]
Heh, yeah that'd be one way of doing it... But it'd be great if each album under Spotify's Your Music could automatically get ported over as an album (as in "Add to My Library") in Google Music.

**Comment by Raynes** [comment]
Yeah, I was planning on writing a project to do exactly that. This will get me close enough though, but it'd be so cool if I could sync albums and tracks and not just playlists. I don't use playlists except for offline stuff for driving.

**Comment by jdrews** [comment]
I would be very interested in this feature. Particularly Albums support.
